sdk/modelsrepository/Azure.Iot.ModelsRepository/src/StandardStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.ModelsRepository
+{
+    internal class StandardStrings
+    {
+        public const string GenericResolverError = "Unable to resolve \"{0}\". ";
```

Comment:
Oh, I thought (when seeing code that referenced) this was a resx class.

I'd say either keep the string literals inline (assuming they aren't shared), or put them in a resx.